packages/components/psammead-headings/package.json

```diff
@@ -1,11 +1,11 @@
 {
```

Comment:
You'll need to add the `package-lock.json` changes too, to this PR. (so the version number matches that in the `package.json`)